### DISCUSSION TITLE
pocketbase REST API GET filter for two conditions AND

### DISCUSSION THREAD

**Discussion by Nymuxyzo** [pocketbase/pocketbase#4835] [Q&A]
I would like to filter data records that have been deleted and archived. If I use an OR linkage it is working, but not with AND(&&)

ERROR GET /api/collections/tasks/records?filter=(deleted=null&&archived=false)&fields=*,expand.assigned.name,expand.customer&expand=assigned,customer&sort=due_date
└─ Something went wrong while processing your request.
   └─ invalid formatted group - missing 1 closing bracket(s)

**Comment by ganigeorgiev** [comment]
The query parameters must be properly URL encoded (the SDKs do this automatically).

```
/api/collections/tasks/records?filter=%28deleted%3Dnull%26%26archived%3Dfalse%29&fields=%2A%2Cexpand.assigned.name%2Cexpand.customer&expand=assigned%2Ccustomer&sort=due_date
```

**Comment by Nymuxyzo** [comment]
Thank you for your quick reply
